FannyAndersson/PayWay
# Feature/user context

Check UserContext!
It works when user already logged in and reload App as well as user logs in on /login page
How to test:

logout

- navigate to http://localhost:3000/ - you will be redirected to /login
- log in - you will be redirected to / (main page, there you can see user's name and menu)
- navigate to http://localhost:3000/login - you will be navigated to / (main page)
